external-api/src/utils/swagger.util.ts

```diff
@@ -69,6 +69,10 @@ export const swaggerDefinition = {
       name: 'Notify',
       description: 'GOV.UK API',
     },
+    {
+      name: 'Bank Holidays',
+      description: 'GOV.UK API and locally stored data',
```

Comment:
Could be useful to specify that the local data is a backup so we understand its use, so something like
```
description: 'GOV.UK API and locally stored backup data',
```